### ISSUE TITLE
Tree tracking? 

### ISSUE THREAD

**Issue by ShotgunSpoon** [zeyad-mansour/lunar#173]
I made an issue asking how to fix it tracking trees, maybe improving its verticle sight of players, etc. Soemoen told me to chang ethe settings, but what settinsg do I change. What code do I modify, etc?

**Comment by ShotgunSpoon** [comment]
Can someone plase tell me what settinsg to change to stop it from tracking trees. Or, can someoen give me a version that will not aim for the head of a player? I don't want to hit head shots, sence I am hitting mostly head shots and it looks suspicious.

**Comment by Reddonan** [comment]
Lol you are lucky , I can't even get it to work . But you can watch octoexploitz on YouTube, he made a better version of lunar🙂

**Comment by Reddonan** [comment]
If you can help me with installation ,that would be nice

**Comment by ShotgunSpoon** [comment]
I can't join that discord server, so if someone could zip that up and send it to me that'd e nice. Also telling me why I should use it instead of lunar that'd be great, sence I'm blind using it and I can't read text on screens in YouTube videos so I know nothin about it.

**Comment by ShotgunSpoon** [comment]
@Reddonan  I can help you with setup, but you need an nividia GPU. Intigrated intel cards work as well, but obvious not as wel. A nvidia geForce gtx 1000> is going to work the best. If you have an AMD GPU, in my experience, it just can't run Torch and crashes instantly. Add me on discord shotgunSpoon#6070 and I can help you.

**Comment by Reddonan** [comment]
Ok thanks I have an rtx 3060 ti

**Comment by ShotgunSpoon** [comment]
ok, i have a 3060 so I know it will work well for you. Also, can you send me octoclient or a working discord link. ALl of the youTube links are broken.

**Comment by ShotgunSpoon** [comment]
@Reddonan  can you send me octoexploit? Also, just did pip install -r requirements.txt in the folder.

**Comment by Reddonan** [comment]
I don't have octoclient ,the discord link is expired

**Comment by Reddonan** [comment]
Is lunar still working?

**Comment by ShotgunSpoon** [comment]
Yes, it still works. In my experience, with all of the changes you have to make, the client is not worth it. Lunar is enough for what I need.On Jun 17, 2022, at 1:41 PM, Reddonan ***@***.***> wrote:﻿
Is lunar still working?

**Comment by Reddonan** [comment]
Are you on Instagram or snap chat

**Comment by Reddonan** [comment]
Discord doesn't work with me

**Comment by ShotgunSpoon** [comment]
I am on neither. I am blind and neither work well for me. That is why I use discord. What do you need, I can help you as much as  possible from here.

**Comment by Reddonan** [comment]
Your are blind? Sorry about that

**Comment by Reddonan** [comment]
I'll try my discord again

**Comment by Reddonan** [comment]
What's your epic username

**Comment by ShotgunSpoon** [comment]
ShotgunSpoon is my epic username. Don't worry about it, bro.

**Comment by Reddonan** [comment]
I'll add you

**Comment by Reddonan** [comment]
If you blind how do you text😶

**Comment by Reddonan** [comment]
Do you need to train it to work effectively

**Comment by Reddonan** [comment]
Hey

**Comment by ShotgunSpoon** [comment]
I'll answer your questions one at a time. SOrry I didn't see these until now. 
* I can text and use git hub, because there is types of software called screen readers that wil read things like web pages and windows apps outloud. Games, however, have to be made specifically to communicate with them, something Fortnite hasn't done.
* You don't have to train it to work effectively, however the more powerful the GPU, the better it'll be. To get the best experience, switch from direct X 11 to performance mode, and if possible, turn your settings down as much as possible and get your game and Lunar to run at the same FPS, preferably 60 in game and close to it on Lunar.

**Comment by Reddonan** [comment]
I sent you a request

**Comment by Reddonan** [comment]
@siiinteiii

**Comment by ShotgunSpoon** [comment]
My main account, ShotgunSpoon, was banned for "cheating in a competetive
event" however they are letting me play on alternate accounts. i am
currently using the account ShotgunGoPop instead. Add me on there as that
is the account I will be using until January 8th, 2023. I sent you a
request on that account. Thank you! This should be fun!

**Comment by Reddonan** [comment]
Hey send a request. I'm currently on ps4 my PC is bad. So I just want to see how good lunar is .maybe we play a game?

**Comment by Reddonan** [comment]
Hey

**Comment by ShotgunSpoon** [comment]
dk your mic was really bad and I couldn't hear or understand you when I joined you awhile back. IDK if you fixed that but until that is fixed we can't play duos sense I need perfect communication.

**Comment by ShotgunSpoon** [comment]
What is your epic game name?

**Comment by Reddonan** [comment]
Idk wym

**Comment by Sultanv** [comment]
Does it still work for you?
